src/autocomplete.js

```diff
@@ -523,6 +526,9 @@ export default class Autocomplete extends Component {
             const showFocused = focused === -1 ? selected === index : focused === index
             const optionModifierFocused = showFocused && hovered === null ? ` ${optionClassName}--focused` : ''
             const optionModifierOdd = (index % 2) ? ` ${optionClassName}--odd` : ''
+            const ariaLabelProp = (this.props.isIosDevice()) ? {
+              'aria-label': `${option} ${index + 1} of ${options.length}`
```

Comment:
This makes me a little nervous, especially if the user is using a suggestion template, as what is being displayed to the user might not match `${option}`, and we're overriding it by providing a label.

I'm also wondering whether Voiceover supports `aria-posinset` / `aria-setsize` as suggested in #277 as this would remove the need to provide a label.

Alternatively, could we explore associating the position and number of results available using `aria-describedby` or any another method?

Reply:
I'll investigate the use of the ARIA attributes as an alternative.
Adam Liptrot (one of the other FE guys here) also made the same suggestion.

If that doesn't yield results, I can possibly use the same resolving function that is being used for the displayed value, to remove the possibility of discrepancy? Will look into.

Reply:
On testing, the `aria-posinset` and `aria-setsize` attributes unfortunately don't help the situation on iOS, ..but they **do** resolve the stale count problem in NVDA (as per the issue you've linked). So the extra attributes are now added.

In terms of the iOS touch/swipe I've settled on an alternative approach, conditionally appending a (visually hidden) `span` element to each `li` option element. The span contains the position info, e.g. `2 of 6`, and caters for the user defined template scenario by appending to the `dangerouslySetInnerHtml`  content - which is already populated from the `templateSuggestion()` resolver function.
Tested on iPhone - you can hit the 'custom results' instance on the test page on my github.io [here](https://markhunter27.github.io/accessible-autocomplete/examples/) to observe.

I did first try an `aria-describedby` approach. I rendered a second `ul` of description elements each referenced using the attribute, but saw some unusual behaviour - always on the first element of the list, where iOS and NVDA both announce `1 of 1` despite the content showing e.g. `1 of 3` in the chrome/FF inspector. It could be that an earlier render() invocation is setting the attribute before the options data is populated and the screen reader isn't budging once it has a value in it's hands. I tried a few different variations, and then moved to the suffix approach which felt tidier all round.